### ISSUE TITLE
how put the same toogle switch in a Hue-Like Light Card?

### ISSUE THREAD

**Issue by xitters** [Nerwyn/material-rounded-theme#33]
HI nice theme!
I have a question, how put the same toogle switch in a Hue-Like Light Card? its posible? Thanks im very noob in HA.
![Sin título](https://github.com/user-attachments/assets/4f960b53-2db3-4832-bc06-7af70f14e5e9)

type: custom:hue-like-light-card
entities:
  - light.aplique_sala
  - light.hue_go_1
  - light.lampara_pie
  - light.spot_sala
title: Sala
offColor: theme-color
hueScreenBgColor: theme-color
offShadow: false
hueBorders: false
icon: mdi:sofa

**Comment by Nerwyn** [comment]
I'll have it's shadow-root path to [this file](https://github.com/Nerwyn/material-rounded-theme/blob/main/src/common/card.yaml).

**Comment by xitters** [comment]
> I'll have it's shadow-root path to [this file](https://github.com/Nerwyn/material-rounded-theme/blob/main/src/common/card.yaml).

How i put this in this code?:
type: custom:hue-like-light-card
entities:

light.aplique_sala
light.hue_go_1
light.lampara_pie
light.spot_sala
title: Sala
offColor: theme-color
hueScreenBgColor: theme-color
offShadow: false
hueBorders: false
icon: mdi:sofa

**Comment by Nerwyn** [comment]
You can't. It's the theme modifying the styles of the toggle switch to match the Material Design 3 style. I can add it in a patch version.

**Comment by xitters** [comment]
> You can't. It's the theme modifying the styles of the toggle switch to match the Material Design 3 style. I can add it in a patch version.

Oh I see, sorry im noob. 
That would be great, waiting for the patch. Thank you so much

**Comment by Nerwyn** [comment]
Released [2.1.3](https://github.com/Nerwyn/material-rounded-theme/releases/tag/2.1.3).

The switch in `hue-like-light-card` is positioned absolutely, so you have to use card-mod to shift it up otherwise it'll be too close to the slider.

```yaml
card_mod:
  style: |
    ha-switch {
      top: 12px !important;
    }
```

**Comment by xitters** [comment]
> Released [2.1.3](https://github.com/Nerwyn/material-rounded-theme/releases/tag/2.1.3).
> 
> The switch in `hue-like-light-card` is positioned absolutely, so you have to use card-mod to shift it up otherwise it'll be too close to the slider.
> 
> ```yaml
> card_mod:
>   style: |
>     ha-switch {
>       top: 12px !important;
>     }
> ```

Excellent!!! Thanks!
![Captura de pantalla 2024-09-17 195559](https://github.com/user-attachments/assets/49f202bd-75a0-41ea-b60f-2d53035eaf22)
